doc/cephfs/standby.rst

```diff
@@ -3,53 +3,50 @@
 Terminology
 -----------
 
-A Ceph cluster may have zero or more CephFS *file systems*.  CephFS
-file systems have a human readable name (set in ``fs new``)
-and an integer ID.  The ID is called the file system cluster ID,
-or *FSCID*.
+A Ceph cluster may have zero or more CephFS *file systems*.  Each CephFS has
+a human readable name (set at creatiopn time with ``fs new``) and an integer
+ID.  The ID is called the file system cluster ID, or *FSCID*.
 
-Each CephFS file system has a number of *ranks*, one by default,
-which start at zero.  A rank may be thought of as a metadata shard.
-Controlling the number of ranks in a file system is described
-in :doc:`/cephfs/multimds`
+Each CephFS file system has a number of *ranks*, numbered beginning with zero.
+By default here is one rank per filesystem.  A rank may be thought of as a
```

Comment:
```suggestion
By default there is one rank per filesystem.  A rank may be thought of as a
```

Reply:
Nice catch, thanks. I'm also splitting `filesystem`, since `file system` is the prevailing convention here.